sklearn/metrics/scorer.py

```diff
@@ -514,6 +514,13 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
 log_loss_scorer = make_scorer(log_loss, greater_is_better=False,
                               needs_proba=True)
 log_loss_scorer._deprecation_msg = deprecation_msg
+# Currently brier_score_loss don't support the shape of result
```

Comment:
If ``brier_score`` is the odd one in not supporting the standard predict_proba shape, we might want to fix it there instead? I'm not entirely certain, though...